### DISCUSSION TITLE
How to install and run osgearth demo?

### DISCUSSION THREAD

**Discussion by mondayboy1** [pelicanmapping/osgearth#2300] [Support Q&A]
Hi, I am a beginner for C++. I want to install osgearth and get some trouble.I referred to this document(http://docs.osgearth.org/en/latest/install.html) and install it through "vcpkg install osgearth:x64-windows", but I don't know what "Then you must set up your CMake configuration to point at the installed libraries" exactly mean. I skipped it and writed the "Sample main.cpp" in vs2022 and return a error `Error reading file .osgearth_engine_rex: file not handled
[osgEarth]* WARNING: Failed to load terrain engine driver for "rex"
[osgEarth]* FAILED to create a terrain engine for this map`. So how do I install osgearth correctly?

**Comment by gwaldron** [comment]
When you ruin `vcpkg install` it installs the osgEarth libraries somewhere, and those libraries (.dll files) need to be in your system path when you run an executable.
By default, vcpkg installs binaries in `[vcpkg_dir]/installed/x64-windows/bin`. Add that to your path.

**Reply by mondayboy1** [reply]
Thanks, I will try it.
